packages/contracts-bedrock/foundry.toml

```diff
@@ -52,6 +52,7 @@ fs_permissions = [
   { access='read-write', path='./.testdata/' },
   { access='read', path='./kout-deployment' },
   { access='read', path='./test/fixtures' },
+  { access ='read', path='./scripts/hash-onion'}
```

Comment:
This is needed to allow Foundry to read the tokens json files on the `hash-onion` folder